src/Model/Resolver/GetCartForCustomer.php

```diff
@@ -120,7 +121,8 @@ public function resolve(
             [
                 'items' => $itemsData,
                 'tax_amount' => $tax_amount,
-                'discount_amount' => $discount_amount
+                'discount_amount' => $discount_amount,
+                'is_virtual' => !!$cart->getIsVirtual()
```

Comment:
According to PHPDoc getIsVirtual is returning bool, however in sources I can see it is coercing value to int. Worth to leave a comment for this coercion, let's prefer (bool) $variable over !!$variable

Reply:
Sure!